pkgs/servers/misc/navidrome/default.nix

```diff
@@ -20,13 +26,17 @@ stdenv.mkDerivation rec {
   installPhase = ''
      mkdir -p $out/bin
      cp navidrome $out/bin
+     mkdir -p $out/lib/systemd/system
```

Comment:
Ah, I see. We don't compile from source. Presumably because of `node`.

Reply:
Looks like it. Unfortunately with this only x86_64 is supported.

Reply:
I think I've seen someone build from source **and** use the binary release just for the `node` assets before :thinking: Sure, that is a bit hacky, but might be worth investigating in a future PR.